src/js/api/config.js

```diff
@@ -28,6 +28,7 @@ export const Defaults = {
     enableShortcuts: true,
     height: 360,
     intl: {},
+    item: 0,
```

Comment:
Thoughts on replacing `item` with `index` or something similar? When I think of `item` in the player, it's usually always an object.

Reply:
I would like that better too, but I do not want to change any model property names until the next major version.

Reply:
That's why I also push for us to use `PlaylistItem` as the type name or part in any API method related to a playlist item object. `item` or event `Item` is vague and can be anything.